### PR TITLE
10242 new form partial name

### PR DESCRIPTION
**Pull request by TJM**
Updated for Foreman 1.8 changes

### CODE REVIEW

File: lib/foreman_digitalocean/engine.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@ class Engine < ::Rails::Engine
 
     initializer 'foreman_digitalocean.register_plugin', :after => :finisher_hook do
       Foreman::Plugin.register :foreman_digitalocean do
-        requires_foreman '> 1.4'
+        requires_foreman '> 1.8'
```
Comment (mmoll) [inline lib/foreman_digitalocean/engine.rb]: '>= 1.8'

**Comment by TJM** [comment]
My testing against 1.8-RC3 was successful. with the "released" version I got a "missing template" error, with this version it works. Does anyone else need to look at this? @mmoll ?

File: lib/foreman_digitalocean/engine.rb (hunk 1 above)
Comment (TJM) [inline lib/foreman_digitalocean/engine.rb]: my bad.

**Comment by domcleal** [comment]
@tstrachota might be able to double check it against his core changes

**Comment by tstrachota** [comment]
I confirm the template path is correct for 1.8

**Comment by TJM** [comment]
I feel like breaking compatibility with 1.7.x and older seems silly over this one file. Are Symlinks strictly forbidden? Is there a way to "include" one template in another?

**Comment by mmoll** [comment]
it's absolutely no problem.

**Comment by mmoll** [comment]
(to "break" things here)

**Comment by domcleal** [comment]
I have no idea if symlinks will work in a gem, but I think you could do `<%= render :partial => 'compute_resources_vms/form/digitalocean/base' %>` in _digitalocean.html.erb.  You might also need to do `:locals => { ... }` and pass any variables used (like `@compute_resource`) through.

**Comment by TJM** [comment]
OK, well in here it sounds feasible, but in IRC I was convinced otherwise. I think I will let this one stand as is, and if I get the http://projects.theforeman.org/issues/10109 fixed, I will just cherrypick it back to the foreman-1.7 branch and release 0.1.1 or whatever. Not sure how to handle this, release wise, it can't be released till foreman 1.8 is released, guess people will have to do something tricky?

**Pull request merged**